xml/registry.rnc

```diff
@@ -478,6 +480,7 @@ InterfaceElement =
     Enum |
     element command {
         Name ,
+        attribute requires { text } ? ,
```

Comment:
I flip-flopped on whether this should be a schema change or an XML change.  I landed on schema change because the single use of this pattern does seem valid.

It is for `clEnqueueMigrateMemINTEL` within `cl_intel_unified_shared_memory` which `requires="CL_VERSION_1_2"`.  In other cases it's the entire extension that requires an OpenCL version, but this instance is specified at a finer granularity.

Reply:
Yeah, I remember discussing various options for `clEnqueueMigrateMemINTEL` and settling on this as the best option, though I can't find the discussion in an issue so it may have been in a PR comment.  I agree this could be a useful pattern going forward for other extensions, as well.

Reply:
Can't see a discussion on this specifically, but [here](https://github.com/KhronosGroup/OpenCL-Headers/pull/65#issuecomment-630501006) you say you settled on this to get things un-stuck.
And the `requires="CL_VERSION_1_2"` didn't exist on `clEnqueueMigrateMemINTEL` before #290 - checked with git blame.